src/components/Tooltip.js

```diff
@@ -9,11 +9,22 @@ class Tooltip extends Component {
         super(props);
     }
 
+    /**
+     * @author Xiaheng Chen
+     * 
+     * @returns {string}
+     */
+    tooltipPositionClass() {
+        const direction = this.props.direction;
+
+        return direction ? `fs-tooltip-message--${direction}` : 'fs-tooltip-message--right';
```

Comment:
We could use a [Nullish coalescing](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing_operator) operator here to make the logic simpler.